app/views/support/Dates.scala

```diff
@@ -0,0 +1,130 @@
+package views.support
+
+import com.github.nscala_time.time.Imports._
+import org.joda.time.{PeriodType, Instant, Interval}
+import org.joda.time.format.PeriodFormat
+import play.twirl.api.Html
+
+object Dates {
+
+  val humanPeriodFormat = PeriodFormat.getDefault()
+
+  val YearMonthDayHours = PeriodType.yearMonthDayTime.withMinutesRemoved.withSecondsRemoved.withMillisRemoved
+
+  implicit
+
+  class RichPeriod(period: Period) {
+    lazy val pretty = period.withMillis(0).toString(humanPeriodFormat)
+  }
+
+  implicit
+
+  class RichDateTime(dt: DateTime) {
+    lazy val pretty = prettyDate(dt)
+    lazy val prettyWithTime = prettyDateWithTime(dt)
+  }
+
+  implicit
+
+  class RichLocalDate(ld: LocalDate) {
+    val dt = ld.toDateTimeAtCurrentTime()
+    lazy val pretty = prettyDate(dt)
+    lazy val prettyWithTime = prettyDateWithTime(dt)
+  }
+
+  implicit
+
+  class RichInstant(dt: Instant) {
+    lazy val pretty = prettyDate(new DateTime(dt))
+    lazy val prettyWithoutYear = prettyDateNoYear(new DateTime(dt))
+    lazy val prettyWithTime = prettyDateWithTime(new DateTime(dt))
+    lazy val prettyNoYearWithTime = prettyDateNoYearTime(new DateTime(dt))
+
+    def pretty(includeTime: Boolean): String = if (includeTime) prettyWithTime else pretty
+
+    def prettyWithoutYear(includeTime: Boolean): String = if (includeTime) prettyNoYearWithTime else prettyWithoutYear
+
+    def isContemporary(threshold: Duration = 24.hours) = new Interval(dt - threshold, dt + threshold).contains(DateTime.now)
+  }
+
+  implicit
```

Comment:
I copied this file direct from membership-frontend. We're planning to put it into a shared library, so will tidy up the styling at that point.